src/mqt/bench/benchmark_generation.py

```diff
@@ -41,6 +43,7 @@ def _get_circuit(
     benchmark: str | QuantumCircuit,
     circuit_size: int | None,
     random_parameters: bool = True,
+    **kwargs: Any,
```

Comment:
It all depends on what we want here, whether `Any` is just fine here.
If we really want to permit arbitrary kwargs, then `Any` is just fine and a simple `noqa` may be added to all occurrences.
If we only want to permit dedicated, known options, then we might go the route of https://github.com/munich-quantum-toolkit/qcec/blob/549305f2e1705da7eab193a454456ae0ec1dca78/python/mqt/qcec/verify.py#L41 with a typed dictionary that just contains the seed for now)